pylintrc

```diff
@@ -367,7 +367,7 @@ class-attribute-rgx = ([A-Za-z_][A-Za-z0-9_]{2,40}|(__.*__))$
 inlinevar-rgx = [A-Za-z_][A-Za-z0-9_]*$
 good-names = f,i,j,k,db,ex,Run,_,__
 bad-names = foo,bar,baz,toto,tutu,tata
-no-docstring-rgx = __.*__$|test_.+|setUp$|setUpClass$|tearDown$|tearDownClass$|Meta$
```

Comment:
You should check in the pylintrc_tweaks file that generated this.

Reply:
Also, should this be applied instead to edx-lint? Or is this a standard that only applies to edx-platform?

Reply:
oh good call about applying it to edx-lint.

Reply:
ah, good point on making the change in edx-lint.
I expect this might be somewhat controversial, and would be nice to keep consistent org-wide.